cmake/Corrosion.cmake

```diff
@@ -314,47 +322,68 @@ function(corrosion_import_crate)
         "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_FILES_DIRECTORY}/corrosion/${toml_dir_name}.dir/cargo-build.cmake"
     )
 
-    if (CMAKE_VS_PLATFORM_NAME)
-        set (_CORROSION_CONFIGURATION_ROOT --configuration-root ${CMAKE_VS_PLATFORM_NAME})
-    endif()
+    if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19.0)
```

Comment:
I guess you probably wanted the `_generator_gen_cmake()` in the true branch here? Or am I missing something

Reply:
It's an embarrassing mistake, thanks.